packages/web/src/scss/components/TextArea/index.html

```diff
@@ -89,7 +92,9 @@ <h2 class="docs-Heading">V1 Feature &mdash; new disabled state</h2>
 
     <div class="TextArea TextArea--disabled">
       <label for="textareaDisabledFilledFeature" class="TextArea__label TextArea__label--required">Label</label>
-      <textarea id="textareaDisabledFilledFeature" class="TextArea__input" placeholder="Placeholder" disabled>Filled</textarea>
+      <textarea id="textareaDisabledFilledFeature" class="TextArea__input" placeholder="Placeholder" disabled>
```

Comment:
```suggestion
      <textarea id="textareaDisabledFilledFeature" class="TextArea__input" placeholder="Placeholder" disabled>
          Filled
      </textarea>
```

Reply:
nope, you don't want that, this adds whitespace inside the textarea

Reply:
Ah! Got it.
And what about closing tag? Is it possible to let the whole name on the one line, or Prettier wouldn't let us?

Reply:
I think we don't. Because this is unwanted behaviour and we try to avoid that. But on the other hand, it is native behaviour, so I don't see reason to mention in the docs.